js/repeater-list.js

```diff
@@ -335,6 +335,9 @@
 					rowData: clickedRow.data('item_data')
 				});
 			}
+			if (selectedObj.length === 1) {
+				selectedObj = selectedObj[0];
+			}
```

Comment:
row level actions at line 306 above passes a single object to `list_getActionItems`.

change the condition to check if `selected` is an array or an obj. `if (!<obj>)` do for loop. `else selectedObj = selected`